### PR TITLE
peribolos: bump timeout to 210m for CI jobs

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
Peribolos CI jobs [have been failing](https://kubernetes.slack.com/archives/C01672LSZL0/p1712182376329389) for a few months now due to (i) timeout issues and (ii) oauth token scope issues.

PR https://github.com/kubernetes/k8s.io/pull/6667 & https://github.com/kubernetes/test-infra/pull/32410 added a new dedicated github token for Peribolos CI jobs, potentially fixing (ii).

This PR is bumping the timeout duration for the Peribolos CI jobs by 30 min (new timeout - 210m or 3.5 hrs), to try fixing (i).

cc: @cblecker @MadhavJivrajani

### CODE REVIEW

File: config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml
Comment (MadhavJivrajani) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml:16]: Can you pls add a TODO comment saying this needs to be bumped down once we figure out what's happening. Because 3 hours should be sufficient time imo.

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
Just one comment, LGTM after that!

**Comment by ameukam** [comment]
/lgtm
/approve
/hold

Remove the hold when ready.

**Comment by cblecker** [comment]
/close
see https://github.com/kubernetes/org/pull/4876

**Pull request closed without merging**
